packages/mint-components/src/components/tax-and-cash/sqm-user-info-form/sqm-tax-document-step-1.feature

```diff
@@ -0,0 +1,57 @@
+@owner:andy
+@author:andy
+
+Feature: Tax Form Step One
+
+
+	Background: A user is prompted to enter their personal and initial tax information
+		Given a user is on the First step of the tax form
+
+
+    @minutia
+    Scenario Outline: A user fills out Tax Form Step One with the required information
+    Given the user is on Tax Form Step One
```

Comment:
I think your background covers this given, you can probably just start it with the "when"